src/styles/components/navbar.scss

```diff
@@ -79,31 +53,9 @@
       transition-property: transform;
```

Comment:
Why is the part with:
```
transition: 0.75s;
transition-delay: 0.01s;
transition-property: transform;
```

written here again? And what's the point of ` 0.01s;` delay?